pom.xml

```diff
@@ -850,6 +856,13 @@
                 <version>${aws.sdk.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <version>${lombok.version}</version>
+                <scope>provided</scope>
+            </dependency>
```

Comment:
Move this out from `<dependencyManagement` and move it into `<dependencies>`

Reply:
Done.